src/app/components/calendar/calendar.component.html

```diff
@@ -0,0 +1,3 @@
+<nz-calendar >
+
+  </nz-calendar>
```

Comment:
a component containing just a component?

Reply:
I still need to add logic, but it will be left for last